modules/react/badge/stories/examples/Basic.tsx

```diff
@@ -21,7 +21,7 @@ const inverseBackground = createStyles({
   padding: system.space.x4,
 });
 
-export function Basic() {
+export const Basic = () => {
```

Comment:
Examples need to be named exports with the arrow function in order for them to be properly converted to default exports in [`build-mdx`](https://github.com/Workday/canvas-kit/blob/master/modules/docs/utils/build-mdx.js#L105) (they need to be default exports to work properly with the site).